src/ScottPlot4/ScottPlot.Cookbook/Recipes/Axis.cs

```diff
@@ -221,4 +221,31 @@ public void ExecuteRecipe(Plot plt)
             plt.XAxis.DateTimeFormat(true);
         }
     }
+    class TicksDateTimeSignal : IRecipe
+    {
+        public ICategory Category => new Categories.Axis();
+        public string ID => "ticks_dateTime_signal";
+        public string Title => "Plotting DateTime Data on a Signal Plot";
+        public string Description =>
+            "DateTime can be displayed on the horizontal axis of a signal plot by " +
+            "using the sample rate to set the time interval per data point, and then " +
+            "setting the OffsetX to the desired start date.";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            // create data sample data
+            double[] ys = DataGen.RandomWalk(100);
+
+            // Since the integral part of an OA date represents days,
+            // need to sample at 24 hours * 60 minutes * 60 seconds to
+            // treat data points as 1 per second
+            var signalPlot = plt.AddSignal(ys, 86400);
```

Comment:
Can we break this into a few lines? Rather than a big comment, it may be more instructive to start with a DateTime or TimeSpan object... something like


```cs
TimeSpan ts = TimeSpan.FromSeconds(1); // time between data points
double sampleRate = (double)ts.Ticks / TimeSpan.TicksPerDay;
var signalPlot = plt.AddSignal(ys, sampleRate);
```

I think it's worth being verbose here because this is the confusing bit

Reply:
Yes even better

Reply:
Corrected 
![image](https://user-images.githubusercontent.com/1486613/198751864-781282c1-e97e-49ad-a02b-98575e5d795f.png)